server/routes/induction/common/personalInterestsController.ts

```diff
@@ -31,8 +37,8 @@ export default abstract class PersonalInterestsController extends InductionContr
 
 const toPersonalInterestsForm = (inductionDto: InductionDto): PersonalInterestsForm => {
   return {
-    personalInterests: inductionDto.personalSkillsAndInterests?.interests.map(interest => interest.interestType) || [],
-    personalInterestsOther: inductionDto.personalSkillsAndInterests?.interests.find(
+    personalInterests: inductionDto.personalSkillsAndInterests?.interests?.map(interest => interest.interestType) || [],
```

Comment:
This change, and the next file, fixes a bug (effectively a NPE) where an Induction might not have the relevant data